tests/unit/doc-printer.js

```diff
@@ -46,9 +46,9 @@ test("`printDocToString` should not manipulate docs", () => {
 
 describe("`printDocToString` has linear time complexity at most to print fill()", () => {
   const baseSize = 3_000;
-  const relativeMargin = 0.4;
+  const relativeMargin = 0.6;
   const baseTime = time(makeFill(baseSize));
-  test.each([10_000, 20_000, 40_000])("numWords=%d", (numWords) => {
+  test.each([20_000, 40_000, 60_000])("numWords=%d", (numWords) => {
```

Comment:
Most failed attempt is `10_000`. I replaced it by `60_000` since bigger `numWords` should be more stable.